File: practices/r2dbc-webflux/src/main/java/com/grizz/wooman/webflux/repository/UserR2dbcRepository.java
package com.grizz.wooman.webflux.repository;

import com.grizz.wooman.webflux.common.repository.UserEntity;
import org.springframework.data.r2dbc.repository.R2dbcRepository;

public interface UserR2dbcRepository
        extends R2dbcRepository<UserEntity, Long> {
}

File: practices/coroutine-r2dbc-webflux/src/main/kotlin/com/grizz/wooman/coroutine/repository/UserR2dbcRepository.kt
package com.grizz.wooman.coroutine.repository

import com.grizz.wooman.webflux.common.repository.UserEntity
import org.springframework.data.repository.kotlin.CoroutineSortingRepository

interface UserR2dbcRepository : CoroutineSortingRepository<UserEntity, Long>
